storage/range.go

```diff
@@ -161,9 +161,9 @@ type RangeManager interface {
 	Clock() *hlc.Clock
 	Engine() engine.Engine
 	DB() *client.KV
-	Allocator() *allocator
+	Allocator() *Allocator
```

Comment:
We shouldn't export allocator or splitQueue. I think we can make either the RangeManager interface or the Allocator method unexported instead.